packages/IMAPClient-UI.package/ICFolderDialog.class/instance/emailListMenu..st

```diff
@@ -14,9 +14,11 @@ emailListMenu: aMenu
 		add: ((self selectedEmail flagged) 
 			ifTrue: ['Unflag']
 			ifFalse: ['Mark as flagged'])  
-			action: [self modifyFlag: #flagged];
-		add: (self labelForDeletionAction: self selectedEmail folder folderName)
-			action: #deleteEmail;
-		add: 'Move to ...' subMenu: self folderListMenu.	
+			action: [self modifyFlag: #flagged].
+
+	(self selectedEmail deleted)
```

Comment:
Here being deleted means that the mail is in the trash folder.
You can not delete mails that are in the trash folder, because it would do nothing.
Therefore the option should not be displayed for these mails.
The behaviour was that way before so we wanted to keep it.

Reply:
I think this could be a new discussion. Why shouldn't I be able to delete mails permanently?

Reply:
That is exactly what expunge does... So you want to delete a single mail permanently?

Reply:
I mean, the Typical use-case is: 
User deletes e-mail and the mail is in the trash. 
Then the user deletes the mail from trash and its gone permanently. But currently I cannot remove mails from the trash. This is somehow untypical behavior.

Reply:
To permanently delete Mails in IMAP one has to mark the wanted Mail with the delete Flag. Then the client has to send an EXPUNGE command which permanently deletes all mails in the current folder marked with the delete flag.

From the RFC:
> The EXPUNGE command permanently removes all messages that have the
> \Deleted flag set from the currently selected mailbox.

The current behaviour is that when you delete a message it moves it to the trash and flags it with the deleted flag. So we can't currently do this. Supporting this is out of the scope of this PR.

Reply:
All right!